chain-core/src/state/account.rs

```diff
@@ -203,6 +203,22 @@ impl CouncilNode {
     }
 }
 
+#[derive(Debug, Clone, PartialEq, Eq, PartialOrd, Ord, Encode, Decode)]
+#[cfg_attr(all(feature = "serde", feature = "hex"), derive(Serialize))]
+/// Metadata of a validator
+pub struct CouncilNodeMetadata {
+    /// Name of validator
+    pub name: ValidatorName,
+    /// Current voting power of validator
+    pub voting_power: TendermintVotePower,
+    /// Address of staking account of validator
+    pub staking_address: StakedStateAddress,
+    /// Optional security email address of validator
+    pub security_contact: ValidatorSecurityContact,
+    /// Tendermint consensus validator-associated public key
+    pub tendermint_pubkey: TendermintValidatorPubKey,
```

Comment:
the compilation fails because of encoding/decoding in no_std -- could this be simplified to:
```
CouncilNodeMetadata {
pub node: CouncilNode,
    /// Current voting power of validator
      pub voting_power: TendermintVotePower,
      /// Address of staking account of validator
      pub staking_address: StakedStateAddress,
```

Reply:
We don't need `Encode`/`Decode` implementations for this. Removed.